pkg/util/annotation_test.go

```diff
@@ -25,118 +25,6 @@ import (
 	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
 )
 
-func TestMergeAnnotation(t *testing.T) {
```

Comment:
Why remove this test?

Reply:
When ReplaceAnnotation is replaced with MergeAnnotation, the test method name is not changed. Just update.